sn_cli/src/subcommands/wallet.rs

```diff
@@ -150,32 +147,26 @@ pub(crate) async fn wallet_cmds_without_client(cmds: &WalletCmds, root_dir: &Pat
 pub(crate) async fn wallet_cmds(
     cmds: WalletCmds,
     client: &Client,
-    wallet_dir_path: &Path,
+    root_dir: &Path,
```

Comment:
Hey Roland,

Minor point, but I remember changing this variable from `root_dir` -> `wallet_dir_path`. Why go back to something vague from something specific? The path does refer to the wallet directory does it not?

Cheers,

Chris